website/docs/r/arc_kubernetes_cluster_extension.html.markdown

```diff
@@ -0,0 +1,110 @@
+---
+subcategory: "ArcKubernetes"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_arc_kubernetes_cluster_extension"
+description: |-
+  Manages an Arc Kubernetes Cluster Extension.
+---
+
+# azurerm_arc_kubernetes_cluster_extension
+
+Manages an Arc Kubernetes Cluster Extension.
+
+-> **Note:** Installing and configuring the Azure Arc Agent on your Kubernetes Cluster to establish connectivity is outside the scope of this document. For more details refer to [Deploy agents to your cluster](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-agent-overview#deploy-agents-to-your-cluster) and [Connect an existing Kubernetes Cluster](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli#connect-an-existing-kubernetes-cluster). If you encounter issues connecting your Kubernetes Cluster to Azure Arc, we'd recommend opening a ticket with Microsoft Support.
+
+-> **Note:** An extensive example on connecting the `azurerm_arc_kubernetes_cluster` to an external kubernetes cluster can be found in [the `./examples/arckubernetes` directory within the GitHub Repository](https://github.com/hashicorp/terraform-provider-azurerm/tree/main/examples/arckubernetes).
```

Comment:
We already have these notes on the arc kubernetes resource docs so I would remove these
```suggestion
```